docs/domains/sub-domains.md

```diff
@@ -69,6 +69,9 @@ title: Sub Domains
 |-------------------------------|------------------------------------------------------------------------------------------------------------------------------------|
 | `cdn.bereal.network`          | New CDN running on `MCI Communications Services, Inc. d/b/a Verizon Business`   pointing to `storage.bere.al` Bucket               |
 | `cdn-resize.bereal.network`   | Image optimization CDN																											 |
+| `cdn-c.bereal.network`        | New CDN      																													     |
+| `cdn-goo.bereal.com`          | New CDN      																												         |
+| `cdn-c.bereal.network`        | New CDN 																												             |
```

Comment:
Lines `72` and `74` are repeated

Reply:
yes i will change that also i wanted to add this :
`## moments

## Verified Endpoints

| URL                                                             | Use           	 						  |                                                                                                                                         
|-----------------------------------------------------------------|-------------------------------------------|
| `https://mobile.bereal.com/api/bereal/moments/last/europe-west` | Gets last (eu) Bereal notification time   |

## Requests and Responses

Response

```json
{
		"id": {
			"NywegFScSAG83Fg4rugyI",
		    "startDate":"2024-01-14T10:37:05.108Z",
			"endDate":"2024-01-14T10:39:05.108Z",
			"region":"europe-west",
			"timezone":"Europe/Paris",
			"localTime":"11:37",
			"localDate":"2024-01-14"
}
```
`
but every-times the Github build fails any ideas ?

Reply:
I think the pre-commit action has a panicks. May be worth me having a look when I'm back from the gym.

Worst case we just push it and deal with it another day

Reply:
yes we can push it if you are okay i will add this later on another PR :)

Reply:
Leave it as is, I'll fix the build step and merge it when I'm back

Thanks as always

Reply:
Thanks, and sure, it's always a pleasure to search and add information to this repository. I learn so much.